content/docs/references/cli/_index.md

```diff
@@ -855,13 +858,12 @@ Lists PowerFlex volume infomation for a given tenant on a local cluster
 
 ##### Output
 
-Lists PowerFlex volume infomation for a given tenant on a local cluster. The namespace is the namespace where tenant secret is created. 
+Gets PowerFlex volume infomation for a given tenant on a local cluster. The namespace is the namespace where tenant secret is created. 
 
 ```
-# dellctl volume list --proxy <proxy.dell.com> --namespace vxflexos
-NAME             VOLUME ID          SIZE       POOL     SYSTEM ID          PV NAME          PV STATUS   STORAGE CLASS   PVC NAME                NAMESPACE
-k8s-0325497cd7   a69b554f00000004   8.000000   mypool   636468e3638c840f   k8s-0325497cd7   Released    vxflexos        vol-create-test-cthdf   replication-suite-fe2eac41
-k8s-a0c031582b   a69b555000000005   8.000000   mypool   636468e3638c840f   k8s-a0c031582b   Released    vxflexos        vol-create-test-nqfwz   replication-suite-fe2eac41
-k8s-3908a6954f   a69b555100000006   8.000000   mypool   636468e3638c840f   k8s-3908a6954f   Released    vxflexos        vol-create-test-4flg5   replication-suite-fe2eac41
-k8s-28e4184f41   a69b554e00000003   8.000000   mypool   636468e3638c840f   none             none        none            none                    none
+# dellctl volume get --proxy <proxy.dell.com> --namespace vxflexos
```

Comment:
Output looks good, but can we improve output to show cases where PV (and/or) PVC is missing?

Reply:
@JacobGros Thanks for the review. Committed changes.

Reply:
> Output looks good, but can we improve output to show cases where PV (and/or) PVC is missing?

Reply:
> Output looks good, but can we improve output to show cases where PV (and/or) PVC is missing?

Reply:
> @JacobGros Thanks for the review. Committed changes.

Reply:
> Output looks good, but can we improve output to show cases where PV (and/or) PVC is missing?